dashboard/modules/node/node_head.py

```diff
@@ -146,7 +161,31 @@ async def _update_nodes(self):
             except Exception:
                 logger.exception("Error updating nodes.")
             finally:
-                await asyncio.sleep(node_consts.UPDATE_NODES_INTERVAL_SECONDS)
+                self._node_update_cnt += 1
+                # Until the head node is registered, we update the
+                # node status more frequently.
+                # If the head node is not updated after 10 seconds, it just stops
+                # doing frequent update to avoid unexpected edge case.
+                if (
+                    not self._head_node_registration_time_s
+                ) and self._node_update_cnt < 100:
+                    await asyncio.sleep(0.1)
```

Comment:
Could we make those magical numbers into vars/consts? E.g.  the 10 seconds and the 0.1 second frequency.